docs/install.rst

```diff
@@ -4,13 +4,29 @@
 Installation
 ============
 
-You can install ``nbgitpuller`` from PyPI with ``pip`` in the same
-environment where your jupyter notebook package is installed.
+nbgitpuller can work on any computer, but it is most-commonly used with a JupyterHub.
+By installing nbgitpuller in the user environment for your hub, it means that
+all users will be able to click nbgitpuller links to get the content.
+
+.. admonition:: To set up a JupyterHub
+   :class: tip
+   
+   If you do *not* have a JupyterHub, we recommend trying out `The Littlest
+   JupyterHub <https://tljh.jupyter.org>`_ to set one up.
+   It comes built in with nbgitpuller.
+
+   For more information about JupyterHub, see
+   `the JupyterHub Documentation <https://jupyterhub.readthedocs.io/en/stable/>`_.
+
+You can install ``nbgitpuller`` from PyPI with ``pip``:
 
 .. code:: bash
 
    pip install nbgitpuller
 
+If you're installing nbgitpuller on a JupyterHub, make sure that it is
+installed in the environment that your users have access to.
```

Comment:
```suggestion
```{note}

If you use multiple environments in your JupyterHub, make sure you are 
installing nbgitpuller on a JupyterHub, make sure that the jupyter notebook or
jupyter server process is running from. You can validate this by running
`jupyter serverextension list` - it should have an entry that says `nbgitpuller  enabled`.
```